src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 40 * days;
+  } else if (days >= 3 && days < 7) {
+    return 40 * days - 20;
+  } else {
+    return 40 * days - 50;
+  }
```

Comment:
Доречно було б додати змінну `rentCost`, т.к. значення `40` використовується 3 рази:
```suggestion
  const rentCost = 40;
  
  if (days < 3) {
    return rentCost * days;
  } else if (days >= 3 && days < 7) {
    return rentCost * days - 20;
  } else {
    return rentCost * days - 50;
  }
```

Reply:
agree